orc8r/cloud/deploy/bare-metal-ansible/roles/magma/orc8r/files/orc8r-envdir.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: orc8r-envdir
+type: Opaque
+stringData:
+  CONTROLLER_SERVICES: "CONFIGURATOR,STATE,STREAMER,POLICYDB,METRICSD,CERTIFIER,BOOTSTRAPPER,ACCESSD,OBSIDIAN,DISPATCHER,DIRECTORYD,CTRACED"
```

Comment:
@mpgermano do we need this environment variable anymore after the service mesh changes ?